module/VuFind/src/VuFind/MetadataVocabulary/PluginFactory.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * Metadata vocabulary plugin factory
+ *
+ * PHP version 7
+ *
+ * Copyright (C) University of Tübingen 2019.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Metadata_Vocabularies
+ * @author   Mario Trojan <mario.trojan@uni-tuebingen.de>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development Wiki
+ */
+namespace VuFind\MetadataVocabulary;
+
+use Interop\Container\ContainerInterface;
+
+/**
+ * Metadata vocabulary plugin factory
+ *
+ * @category VuFind
+ * @package  Metadata_Vocabularies
+ * @author   Mario Trojan <mario.trojan@uni-tuebingen.de>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development Wiki
+ */
+class PluginFactory extends \VuFind\ServiceManager\AbstractPluginFactory
+{
+    /**
+     * Constructor
+     */
+    public function __construct()
+    {
+        $this->defaultNamespace = 'VuFind\MetadataVocabulary';
+    }
+    
+    public function __invoke(ContainerInterface $container, $requestedName,
```

Comment:
This __invoke method appears to be copied-and-pasted from AbstractPluginFactory. I think you can delete it and inherit the base version.